tests/unit/utils.py

```diff
@@ -75,6 +76,10 @@ def __init__(self, target_path, data_root, mocks, expected_bugs,
                  expected_issues, sub_root):
         self.sub_root = sub_root
         self.target_path = target_path
```

Comment:
self.target path should be pointing to the scenario so in fact you should be able to check that rather than create a new "scenario_path"

Reply:
The code needs an absolute path to the scenario, so I thought that having an intermediate variable would be a good idea. Should the "target_path" also be set to the absolute path of the scenario?

Reply:
no, it is the target as in https://hotsos.readthedocs.io/en/latest/contrib/testing.html#test-reference so it is a path but relative to a scenario (most of the time it is a filename). But ```os.path.join(self.sub_root, target_path)``` gives you a path relative to ```defs```  e.g. with *target_path* set to "unattended_upgrades.yaml" you would get ```scenarios/system/unattended_upgrades.yaml``` which is the scenario that is expected to exist.

Reply:
So I tried `if not os.path.exists(os.path.join(self.sub_root, self.target_path)):` but it's failing for some test cases, so I still need to use the `DEFS_DIR` (i.e. `if not os.path.exists(os.path.join(DEFS_DIR, self.sub_root, self.target_path)):`) in order to make it work. Am I missing something here?

Reply:
it think the way it is currently is good. scenario path is now ```os.path.exists(os.path.join(DEFS_DIR, self.sub_root, self.target_path))``` which is correct.